File: VB.NET/windows/progress1/progress1/Form1.vb
Public Class Form1
    Dim i As Integer = 1
    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Timer1.Enabled = True
        Label1.Visible = True

    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        Timer1.Stop()
    End Sub

    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        ProgressBar1.Value = i
        i = i + 1
        Label1.Text = "Remaining  " + CStr(100 - i)
        If (i >= 100) Then
            Timer1.Stop()
        End If
    End Sub
End Class

File: CS1/Windows/progress1/progress1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace progress1
{
    public partial class Form1 : Form
    {
        int i=1;
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
             timer1.Enabled = true;
             label1.Visible = true;
        }

        private void timer1_Tick(object sender, EventArgs e)
        {
            progressBar1.Value = i;
            i++;
            label1.Text = "Remaining  " + (100-i);
            if (i >= 100) 
                timer1.Stop();      
        }

        private void button2_Click(object sender, EventArgs e)
        {
            timer1.Stop();
        }

        
       
    }
}